static/app/views/insights/http/views/httpLandingPage.tsx

```diff
@@ -153,10 +212,28 @@ export function HTTPLandingPage() {
       sorts: [sort],
       limit: DOMAIN_TABLE_ROW_COUNT,
       cursor,
+      enabled: !useEap,
     },
     Referrer.LANDING_DOMAINS_LIST
   );
 
+  const domainsListResponse = useEap ? eapDomainListResponse : metricsDomainsListResponse;
+  const {
+    isPending: isDurationDataLoading,
+    data: durationData,
+    error: durationError,
+  } = useEap ? eapDurationDataResponse : durationDataResponse;
+  const {
+    isPending: isThroughputDataLoading,
+    data: throughputData,
+    error: throughputError,
+  } = useEap ? eapThroughputDataResponse : throughtputDataResponse;
+  const {
+    isPending: isResponseCodeDataLoading,
+    data: responseCodeData,
+    error: responseCodeError,
+  } = useEap ? eapResponseRateResponse : responseRateResponse;
+
```

Comment:
This is a lot of duplication 🤔 this if every single page is going to require this much work, that won't look great in the code. Is it possible to tuck this away inside a hook?